software/contrib/lutra.md

```diff
@@ -0,0 +1,59 @@
+# Lutra
+
+Lutra is a re-imagining of [Expert Sleepers' Otterly](https://expert-sleepers.co.uk/otterley.html) module.
+
+Each output channel is a free running LFO with a subtly different clock speed. The spread of clock speeds is controlled
+by `k2` (and optionally `ain` -- see configuration below)
+
+## Configuration
+
+The functionality of `ain` can be configured by creating/editing `/saved_state_Lutra.txt` on the module. This JSON file
```

Comment:
I interpreted things differently; I see the global config files (experimental_config & europi_config) as module-wide settings, while the saved_state files are effectively config files with a per-script focus.  Adding a script-specific setting to e.g. experimental_config feels like overkill, since that setting would never be used anywhere else.

Creating separate saved-state & config files for each script also feels counter-intuitive to me. The saved state _is_ the current configuration of the script. This particular feature is just a sneaky way of adding an extra bit of functionality that otherwise wouldn't be supported.

Reply:
If that's how you are interpreting these features then the documentation for them needs to be improved. The firmware level config files are certainly module wide settings, but individual scripts can also have configuration, see the turing-machine and diagnostic script for examples.

> This particular feature is just a sneaky way of adding an extra bit of functionality that otherwise wouldn't be supported.

My assumptions about why this feature wouldn't otherwise be supported is that the limited number of hardware UI elements on the EuroPi make adding these features difficult with out making it very menu driven. This is explicitly one of the things that the configuration system was meant to address.

Please see the following issue and PRs for details of the motivations for these features. I will open a PR that pulls these details into the documentation.

https://github.com/Allen-Synthesis/EuroPi/issues/163
https://github.com/Allen-Synthesis/EuroPi/pull/220
https://github.com/Allen-Synthesis/EuroPi/pull/185

Reply:
Fair enough. I'll move that one item out of the saved state & create a separate config file for it.